tubemq-client-twins/tubemq-client-go/client/consumer_impl.go

```diff
@@ -223,17 +228,18 @@ func (c *consumer) GetMessage() (*ConsumerResult, error) {
 	rsp, err := c.client.GetMessageRequestC2B(ctx, m, c.subInfo, c.rmtDataCache)
 	if err != nil {
 		log.Infof("[CONSUMER]GetMessage error %s", err.Error())
+		err1 := c.rmtDataCache.ReleasePartition(true, isFiltered, confirmContext, false)
```

Comment:
This way can use like it:
if err := c.rmtDataCache.ReleasePartition(true, isFiltered, confirmContext, false); err != nil {
}

Reply:
Why to ReleasePartition?